platform-hub-api/app/controllers/kubernetes/revoke_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Kubernetes::RevokeController < ApiJsonController
+
+  # POST /kubernetes/revoke
+  def revoke
+    authorize! :manage, :identity
```

Comment:
Revoke functionality is only ever going to be available to the admin. The difference between this and standard token management (via Identity) for the user is that an admin should be able to quickly lookup and revoke a given token without knowing which user that token is associated to. The only action end user can do for the time being is to "claim" their token.